infra/terraform/prod/elb.tf

```diff
@@ -21,86 +20,96 @@ resource "aws_lb" "wikijump_elb" {
     }
 }
 
-resource "aws_lb_target_group" "elb_target_group" {
+resource "aws_lb_target_group" "elb_target_group_80" {
     name        = "wikijump-tg-80-${var.environment}"
     port        = 80
-    protocol    = "HTTP"
+    protocol    = "TCP"
     vpc_id      = aws_vpc.wikijump_vpc.id
+    target_type = "ip"
     health_check {
-        enabled = false
-        path    = "/heartbeat.php"
-        matcher = "200"
+        enabled = true
     }
 }
 
-resource "aws_lb_listener" "elb_listener" {
+resource "aws_lb_target_group" "elb_target_group_443" {
+    name        = "wikijump-tg-443-${var.environment}"
+    port        = 443
+    protocol    = "TCP"
+    vpc_id      = aws_vpc.wikijump_vpc.id
+    target_type = "ip"
+    health_check {
+        enabled = true
+    }
+}
+
+resource "aws_lb_listener" "elb_listener_80" {
     load_balancer_arn       = aws_lb.wikijump_elb.arn
     port                    = 80
+    protocol                = "TCP"
     default_action {
         type                = "forward"
-        forward {
-            target_group    {
-                arn    = aws_lb_target_group.elb_target_group.arn
-            }
-        }
+        target_group_arn = aws_lb_target_group.elb_target_group_80.arn
     }
 }
 
-resource "aws_lb_listener_rule" "cloudfront_header_check" {
-    listener_arn            = aws_lb_listener.elb_listener.arn
-    priority                = 100
-
-    action {
+resource "aws_lb_listener" "elb_listener_443" {
+    load_balancer_arn       = aws_lb.wikijump_elb.arn
+    port                    = 443
+    protocol                = "TCP"
+    default_action {
         type                = "forward"
-        target_group_arn    = aws_lb_target_group.elb_target_group.arn
-    }
-
-    condition {
-        http_header {
-        http_header_name    = "X-CLOUDFRONT-WIKIJUMP-AUTH"
-        values              = [var.cf_auth_token]
-        }
+        target_group_arn = aws_lb_target_group.elb_target_group_443.arn
     }
 }
 
-resource "aws_lb_listener_rule" "fallback" {
-    listener_arn            = aws_lb_listener.elb_listener.arn
-    priority                = 999
+# resource "aws_lb_listener_rule" "cloudfront_forward_80" {
```

Comment:
Commented code.

Reply:
For the record, I'm fine with large commented blocks of code if there is documentation explaining *why* it's commented, e.g. temporarily disabled, broken and needs a certain kind of fix, going to be deprecated, etc.

Commented out code in general makes me concerned it was a temporary development change that ends up in master, without clarity as to why it was commented out.